staging/src/k8s.io/component-base/logs/logs.go

```diff
@@ -47,7 +42,7 @@ var (
 )
 
 func init() {
-	klog.InitFlags(packageFlags)
+	klogflags.Init(packageFlags)
```

Comment:
I'm unclear why this moved to a different package, personally.

Reply:
It's used in `k8s.io/component-base/logs/api/v1` to access the klog settings and determine which logging flags exist, which in turn is used to warn about those that only have an effect in the text format (used to more, now it's only `-vmodule` which isn't supported by zapr).

If it was not moved into a separate leaf package, there would be a circular dependency. Before, `api/v1` directly called `klog.InitFlags`, but that's not correct anymore because only a subset of those flags are active.

Reply:
Perhaps I also need to explain why klogflags is needed and why it is under component-base:
- We cannot simply remove the flags from klog.InitFlags, that would break non-Kubernetes applications that use it.
- We could add a klog.InitKubernetesFlags, but IMHO that is too specific to Kubernetes and belongs more into component-base than klog.